CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+
+### I want to review PRs. How should I do that?
+PR reviews take time especially for larger PRs. Please take that time and do not rush it.
+We suggest that you split up your review into a functional and code review.
+
+#### Functional Review
+For the functional review, check out the PR locally and get it running. In the best case there is a user flow that you can test. Check the browser console and the logs of the server if there are any warnings or errors in the background. 
+Either everything works according to the ticket or something is broken. In both cases, do still a code review. This way you can maybe help finding a bug in the code and also other issues can be resolved at once.
+
+#### Code Review
+Go to the PR in Github and the "Files Changed" Tab. Go through the changes and check if best practises are followed and if you can read the code. 
+Be constructive and do not forget to appreciate the effort.
+
+[More documentation for code reviews in Github you can find here.](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request)
+
+### I want to suggest a task and create a ticket. How should I do that?
+If you want to create a ticket, add it on the bottom of the "Product Backlog" list in our trello. In the best case, write a user story to explain your reasoning. If you can add a to-do list with the proposed development steps and maybe even a rough estimation how much time you guess it would take (See the T-Shirt sizeing for time estimation).
+If you discovered a bug, there is a template in trello to create a bug ticket. Please use it.
+After creating the ticket, tag our product managers to review and prioritize your ticket. 
+
+### I want to suggest changes in our stack / architecture. How do I do that?
+If possible create an Architecture Decision Record (ADR) as a starting point for your discussion. Please check the [ADR template](docs/adr/adr_template.md) for more information!
+
+## Guidelines
+
+- **Start small.** Contributing to a new project can be quite daunting.
+- **Think about the tests first.** Try [test driven development](https://www.browserstack.com/guide/tdd-vs-bdd-vs-atdd) where possible.
+- **Make sure your idea has support.** Before spending a lot of time on a feature, it's worth checking that it is wanted first. Ask around in Slack and our product owners.
+- **Constructive criticism** If you do not like a contribution, feel free to air it. However, before doing so think first about how to improve and propose alternative solutions.
+
+## Development Principles
+
+These are the principles that have guided our development so far. If you're making a major contribution, it may be worth keeping them in mind.
+
+- **Use boring, well-established technology.** Boxtribute needs to keep on working for a long time with minimal work. We don't want to have to rewrite it every year because a technology we are using has gone out of fashion.
```

Comment:
I don't know if I would agree with the term "boring". There are a couple things that we are using that are quite "exciting" (ReactJS, Ariadne, GraphQL). However, all of the technology we use are stable, very high rates of adoption, and high expected long-term potential. 

Reliability > Sexiness / Flashiness is closer to how I would put it

Reply:
Should also say something like "see some of our ADRs if you want an example of what that looks like"

Reply:
changed.